### ISSUE TITLE
Website content overlaps react-hot-loader-error-overlay

### ISSUE THREAD

**Issue by karolisgrinkevicius-home24** [gaearon/react-hot-loader#1202]
### Description

Rendered `react-hot-loader-error-overlay` div element ⚛️🔥😭 is overlapped by existing content possibly because of absence of `z-index` css property.

### Expected behavior

`react-hot-loader-error-overlay` div element has the most possible value of `z-index` property to overlap existing rendered page elements. 

### Actual behavior

Rendered `react-hot-loader-error-overlay` element is overlapped by existing content.

### Environment

```
Mac OS X 10.14.3 (18D109)
Chrome 72.0.3626.121
node@v11.7.0
npm@6.8.0

└── react-hot-loader@4.8.0
```

**Comment by mohsinulhaq** [comment]
I find the overlay very annoying personally. Not sure why it was added.

**Comment by theKashey** [comment]
1. @karolisgrinkevicius-home24  - I will add `z-index` prop to the configuration
2. @mohsinulhaq - it was added to inform you about EVERYTHING IS BROKEN. UNABLE TO CONTINUE.
3. @mohsinulhaq - you may disable it via configuration, or just replace by your own implementation - https://github.com/gaearon/react-hot-loader#setting-global-error-reporter

**Comment by mohsinulhaq** [comment]
@theKashey I understand, you helped me with that in one of my issues. https://github.com/gaearon/react-hot-loader/issues/1153#issuecomment-455334450

But it was quite an involved change than I was expecting. I just wished it was turned off by default.

**Comment by theKashey** [comment]
There is a little problem with decision making and collecting user feedback. Basically - I am here a bit alone.
The idea behind `error overlay` is to reduce __cognitive dissonance__, when you did everything right, and there is no change in a browser, or a blank screen. You like a - _what a heck, table flip, I am out of here!_.
You are expected to notice a problem and update the code - then ErrorOverlay would automatically disappear. Works great with a dual monitor system, or just a single big one.

**Comment by mohsinulhaq** [comment]
I am actually using https://github.com/smooth-code/error-overlay-webpack-plugin for error overlays already and it handles hot reload errors quite gracefully, that's why I didn't find this overlay useful. Anyway, I get your point. 👍

**Comment by mohsinulhaq** [comment]
@theKashey can this also be incorporated in the next release?

**Comment by theKashey** [comment]
Next release took a place an hour ago, so no 😎

Could you be more concrete about what you actually want to be incorporated? I've lost a track a bit.

**Comment by mohsinulhaq** [comment]
![image](https://user-images.githubusercontent.com/7688231/60593665-e1422180-9dc0-11e9-9466-e0d624724deb.png)
Due to no z-index set, it gets overlapped by the other components in the app.
Although I have disabled the overlay, it still used to come on `4.12.0` on first page load, hence I noticed it again.
That issue is fixed in `4.12.1`, but still it would be good to fix the z-index for others.

**Comment by theKashey** [comment]
Ok, so that's another bug - I am going to change conditions to display it, so it would not appear on page load.
Deal?

**Comment by mohsinulhaq** [comment]
no I mean that issue is fixed in 4.12.1, it just needs fix in the z-index

**Comment by theKashey** [comment]
😅Not intentionally.
